### PR TITLE
Persist NativeScript Inspector breakpoints - close #360

### LINKED ISSUE #360: Debugger - load the breakpoints from the previous session
It would be great to load the breakpoints from the previous debugging session.

### PR DESCRIPTION
**Pull request by jasssonpet**

### CODE REVIEW

File: src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m [hunk 1]
```diff
@@ -39,25 +48,42 @@ - (void)applicationShouldReopen:(NSNotification*)notification {
 - (void)setupWebView {
     WebView* webView = (WebView*)self.view;
 
-    JSContext* context = webView.mainFrame.javaScriptContext;
+    // Breakpoints are saved in window.localStorage so make them sandboxed per application
+    NSString* applicationSupportPath = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES)[0];
+    NSString* appName = [[NSBundle mainBundle].infoDictionary objectForKey:(NSString*)kCFBundleNameKey];
+    NSString* localStoragePath = [NSString pathWithComponents:@[applicationSupportPath, appName, @"Local Storage", self->_projectName]];
```
Comment (KristinaKoeva) [inline src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m]: Shouldn't you update the ```self->_projectName``` when the application is reopened ?

Comment (jasssonpet) [inline src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m]: You are correct.

File: src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m [hunk 2]
```diff
@@ -2,26 +2,31 @@
 @import AppKit;
 @import WebKit;
 @import JavaScriptCore;
+#import <WebKit/WebPreferencesPrivate.h>
 #import "InspectorFrontendHost.h"
 
-@implementation ViewController
+@implementation ViewController {
+    NSString* _mainFileName;
```
Comment (KristinaKoeva) [inline src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m]: IMO if we have this as fields, we don't need to have parameterized methods ( ```update``` could just use ```self->_projectName``` ) but it's not that important

**Comment by KristinaKoeva** [comment]
Looks great to me :+1:

**Pull request merged**